src/cell.cpp

```diff
@@ -31,6 +40,36 @@ void Cell::_bind_methods() {
 int Cell::CollisionCount = 0;
 bool immortal = 0;
 
+void Cell::seteq(Cell *otherCell) {
+	// Create new genome. Overwriting the existing one should destruct it
+	_cellGenome = Genome();
+
+	// copy genome of otherCell
+	for(int i=0; i < otherCell->_cellGenome.getSize(); ++i){
+		Gene* gene = otherCell->_cellGenome.getGene(i);
+
+		Gene* newGene = gene->clone();
+		_cellGenome.addGene(newGene);
+	}
+
+	// Copy relevant cell state information
+	_cellState = this->get_node<CellState>("CellState");
+	_cellState->setAlive(otherCell->_cellState->getAlive());
+	_cellState->setLifespan(otherCell->_cellState->getLifespan());
+	_cellState->setHomeostasisNutrientCost(otherCell->_cellState->getHomeostasisNutrientCost());
+	_cellState->setReproductionNutrientCost(otherCell->_cellState->getReproductionNutrientCost());
+	_cellState->setTotalNutrients(otherCell->_cellState->getTotalNutrients());
+	_cellState->setNutrientMaximum(otherCell->_cellState->getNutrientMaximum());
+	_cellState->setHomeostasisEnergyCost(otherCell->_cellState->getHomeostasisEnergyCost());
+	_cellState->setReproductionEnergyCost(otherCell->_cellState->getReproductionEnergyCost());
+	_cellState->setTotalEnergy(otherCell->_cellState->getTotalEnergy());
+	_cellState->setEnergyMaximum(otherCell->_cellState->getEnergyMaximum());
+
+	// Not sure if this makes sense to do
+	_spriteSize = otherCell->getSpriteSize();
+	applyScale(otherCell->getScale());
```

Comment:
Since this is used in reproduction, total nutrients and total energy should be split equally between the two Cells. Also, the area of the each cell should be half that of the parent cell. That is equivalent to multiplying the scale by sqrt(2) / 2 = 0.7071.
```
a1 = pi * r1 ^2
a2 = pi * r2 ^ 2
   = a2 / 2
   = (pi * r1 ^ 2) / 2
r2 ^ 2 = (r1 ^ 2) / 2
r2 = r1 * sqrt(2) / 2
```